resources/assets/uad_lists.json

```
@@ -6491,7 +6491,7 @@
   {
     "id": "com.samsung.android.app.soundpicker",
     "list": "Oem",
-    "description": "Lets you select a sound for alarm/ringtone\n",
+    "description": "Let's you select a sound for alarm/ringtone\n",
```

```suggestion
    "description": "Lets you select a sound for alarm/ringtone\n",
```
Incorrect. "Lets" is correct: https://www.grammarly.com/blog/lets-vs-lets/#:~:text=The%20difference%20between%20let's%20and%20lets&text=Let's%20is%20a%20contraction%20of%20%E2%80%9Clet%20us.%E2%80%9D%20You%20use,to%20allow%20or%20give%20permission.